src/components/HeaderBar/HeaderBar.tsx

```diff
@@ -1,10 +1,40 @@
-import { Navbar, Container } from 'react-bootstrap';
+import { Navbar, Container, ButtonGroup, Button } from 'react-bootstrap';
+import { Link, useHistory } from "react-router-dom";
 
-export const HeaderBar = () => {
+interface Props {
+    walletAddress: string | undefined;
+    logout: () => void;
+    login: () => void;
+}
```

Comment:
Similar comment regarding the above.. We don't need this if we just use the hook here.

Reply:
in this case better to use context instead of custom hooks :)

Reply:
> in this case better to use context instead of custom hooks :)

Correct, clarified in person with Aerim and Jin 👍